contracts/container/contract.go

```diff
@@ -1072,11 +1078,13 @@ func PutReport(cid interop.Hash256, sizeBytes, objsNumber int, pubKey interop.Pu
 			summary.NumberOfObjects += objsNumber
 
 			report := NodeReport{
-				PublicKey:       pubKey,
-				NumberOfReports: 1,
-				ContainerSize:   sizeBytes,
-				NumberOfObjects: objsNumber,
-				LastUpdateEpoch: currEpoch,
+				PublicKey:        pubKey,
+				NumberOfReports:  1,
+				ContainerSize:    sizeBytes,
+				NumberOfObjects:  objsNumber,
+				LastUpdateEpoch:  currEpoch,
+				LastUpdateTime:   currTime,
+				EpochAverageSize: sizeBytes * (currTime - lastEpochTick) / epochDuration,
```

Comment:
Isn't it `sizeBytes * (lastEpochTick + epochDuration - currTime) / epochDuration`?